sbt-app/src/sbt-test/dependency-management/update-sbt-classifiers/build.sbt

```diff
@@ -8,5 +6,88 @@ dependencyResolution := {
   throw new RuntimeException("updateSbtClassifiers should use updateSbtClassifiers / dependencyResolution")
 }
 
-scalaOrganization := "doesnt.exist"
+lazy val root = (project in file("."))
+  .settings(
+    scalaVersion := "2.11.12",
+    scalaOrganization := "doesnt.exist",
+    name := "myProjectName",
+
+    TaskKey[Unit]("checkModuleIdsInUpdateSbtClassifiers") := {
```

Comment:
I made the test stricter and check module ids in the update report.

I am not 100% sure if such tests are considered ok in the current repository.
On the one hand, it can detect regressions and detect obscure bugs.
On the other hand, it can make the maintenance slightly less convenient - this expected data will be need to be updated once the sbt internal classpath changes. Also, it might be not obvious to detect regressions - "which entries are considered ok in the list?".

I leave the final decision to the project maintainers.